### ISSUE TITLE
Error: Fata Exception : Crash

### ISSUE THREAD

**Issue by batical** [mateusz1913/react-native-avoid-softinput#63]
### Environment

Library version: ^2.4.3
OS version: android android 10 

### Affected platforms

- [X] Android
- [ ] iOS

### Current behavior

Trying to display keyboard => crash

```
Fatal Exception: java.lang.NullPointerException: null cannot be cast to non-null type com.facebook.react.uimanager.RootView
       at java.util.Objects.requireNonNull(Objects.java:228)
       at com.reactnativeavoidsoftinput.AvoidSoftInputManager.onSoftInputHeightChange(AvoidSoftInputManager.kt:125)
       at com.reactnativeavoidsoftinput.AvoidSoftInputManager.onSoftInputHeightChange$default(AvoidSoftInputManager.kt:104)
       at com.reactnativeavoidsoftinput.AvoidSoftInputModule.onSoftInputHeightChange(AvoidSoftInputModule.kt:114)
       at com.reactnativeavoidsoftinput.AvoidSoftInputProvider$onSoftInputHeightChange$$inlined$schedule$1.run(Timer.kt:149)
       at java.util.TimerThread.mainLoop(Timer.java:562)
       at java.util.TimerThread.run(Timer.java:512)

### Expected behavior
```

Do not crash

### Reproduction

 ```
 const onFocusEffect = useCallback(() => {
    if (Platform.OS === 'android') {
      AvoidSoftInput.setAdjustNothing()
      AvoidSoftInput.setEnabled(true)
      AvoidSoftInput.setAvoidOffset(0)
    }

    return () => {
      if (Platform.OS === 'android') {
        AvoidSoftInput.setAvoidOffset(0)
        AvoidSoftInput.setEnabled(false)
        AvoidSoftInput.setDefaultAppSoftInputMode()
      }
    }
  }, [])


  const buttonContainerPaddingValue = useSharedValue(0)

  const buttonContainerAnimatedStyle = useAnimatedStyle(() => {
    return {
      paddingBottom: buttonContainerPaddingValue.value
    }
  })

  useSoftInputShown(({ softInputHeight }) => {
    buttonContainerPaddingValue.value = withTiming(softInputHeight)
  })

  useSoftInputHidden(() => {
    buttonContainerPaddingValue.value = withTiming(0)
  })

  useFocusEffect(onFocusEffect)
```

**Comment by mateusz1913** [comment]
Hi @batical , which version of react-native do you use in your project? Is it possible, that you have 0.68 with "new architecture" **enabled**?

**Comment by mateusz1913** [comment]
@batical , at the first glance, it seems to be not handling optional type. Can you check, will following fix work for you?

In AvoidSoftInputManager.kt in line 125

Change:
```
val rootView = getReactRootView(currentFocusedView) ?: (mPreviousRootView as RootView)
```
To (only add '?' to mark `RootView` as optional):
```
val rootView = getReactRootView(currentFocusedView) ?: (mPreviousRootView as RootView?)
```

**Comment by batical** [comment]
Gonna test it now. Keep you inform

**Comment by batical** [comment]
@mateusz1913  seems to work thanks. Will run more test on multiple device through test lab etc

**Comment by mateusz1913** [comment]
Awesome, if it will work fine, would you be interested in making pull request with that one-line fix?

**Comment by mateusz1913** [comment]
Available with v2.4.4 🎉 , thanks @batical !
